src/frontend/frontend/templates/osint_source/osint_source_import.html

```diff
@@ -1,6 +1,7 @@
 
 
 <div class="p-5">
+  {% include 'partials/admin_form_error.html' %}
```

Comment:
Good idea using the normal error partial, that obsoletes the `result` my suggestion:


```jinja
<div class="p-5" id="osint-source-import">
  <form id="form" hx-trigger="submit" hx-post="{{ url_for('admin.import_osint_sources') }}" hx-target="#osint-source-import" enctype="multipart/form-data">
    <div class="grid">
      <div>
        <label class="label">
          <span class="label-text">Select JSON File</span>
        </label>
        <input type="file" name="file" accept="application/json" class="file-input file-input-bordered w-full" required />
      </div>
    </div>

    <button type="submit" class="btn btn-success w-full mt-5 mb-5">Submit</button>
  </form>

  {% include "partials/admin_form_error.html" %}
</div>
```